projects/noloan2/app/src/main/proto/LAWSUIT.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package hasadna.noloan2;
+
+import "COURT.proto";
+import "SMS.proto";
+
+option java_package = "hasadna.noloan2.protobuf";
+option java_outer_classname = "LawsuitProto";
+
+message Lawsuit {
```

Comment:
Can't it be better to be split to multiple Protos? for the user and the company etc.
Not really sure about this

Reply:
Perhaps. I guess this will be relevant for a more developed stage of the app. Holding spam companies in a DB. Made it more general for now. Easy to adapt later on...